docs/debugging.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Blazor debugging
+author: danroth27
+description: Learn how to debug Blazor apps.
+manager: wpickett
+ms.author: riande
+ms.custom: mvc
+ms.date: 07/25/2018
+ms.prod: asp.net-core
+ms.technology: aspnet
+ms.topic: article
+uid: client-side/blazor/debugging
```

Comment:
Same here with what is also going to be the URL segment. They would prefer `client-side/blazor/debug`.

Reply:
Same deal for the name of the file ... *debug.md*.